Engines/Wine/Engine/Implementation/script.js

```diff
@@ -368,23 +368,21 @@ module.default = class WineEngine {
         let workingContainerDirectory = this.getContainerDirectory(this.getWorkingContainer());
         let distribution = "";
 
-        if (fileExists(workingContainerDirectory)) {
-            const containerConfiguration = configFactory.open(workingContainerDirectory + "/phoenicis.cfg");
+        if (!fileExists(workingContainerDirectory)) {
+            throw new tr('Wine prefix "{0}" does not exist', this.getWorkingContainer());
+        }
 
-            distribution = containerConfiguration.readValue("wineDistribution", "upstream");
-            architecture = containerConfiguration.readValue("wineArchitecture", "x86");
+        const containerConfiguration = configFactory.open(workingContainerDirectory + "/phoenicis.cfg");
 
-            const operatingSystem = operatingSystemFetcher.fetchCurrentOperationSystem().getWinePackage();
+        distribution = containerConfiguration.readValue("wineDistribution", "upstream");
+        architecture = containerConfiguration.readValue("wineArchitecture", "x86");
```

Comment:
Is it still necessary to declare these values above with `let` or could we declare them here using `const`?